api/src/integrations/epic/base-record-utils.js

```diff
@@ -58,6 +60,25 @@ class BaseRecordUtils {
     return documents;
   }
 
+  /**
+   * Deletes any document records linked to by the provided NRPTI record.
+   *
+   * @param {object} nrptiRecord NRPTI record.
+   * @returns
+   * @memberof BaseRecordUtils
+   */
+  async removeDocuments(nrptiRecord) {
+    if (!nrptiRecord || !nrptiRecord.documents || !nrptiRecord.documents.length) {
+      return;
+    }
+
+    const DocumentModel = mongoose.model('Document');
+
+    const documentIds = nrptiRecord.documents.map(id => new ObjectID(id));
+
+    await DocumentModel.deleteMany({ _id: { $in: documentIds } });
+  }
+
```

Comment:
This just deletes the document meta, what about the actual files if we stored them locally/s3 etc?

Reply:
Fair point.  I'm updating it currently so that It fetches the documents, checks if an s3 key is present, and then deletes from s3 in addition to deleting locally.